source/_integrations/home_connect.markdown

```diff
@@ -43,7 +43,9 @@ Note that it depends on the appliance and on API permissions which of the featur
 - OAuth Flow: Authorization Code Grant Flow
 - Redirect URI: `https://my.home-assistant.io/redirect/oauth`
 
-*Important*: 
+*Important*:
+ - **Power on** all your appliances during the integration configuration process, otherwise appliance programs list will be empty.
+ - To update the appliance programs list, you can simply reload the Home Connect integration when an appliance is turned on. If the re-initialization process is not triggered by reload, restart the Home Assistant when an appliance is turned on. 
```

Comment:
```suggestion
 - **Power on** all your appliances during the integration configuration process; otherwise appliance programs list will be empty.
 - To update the appliance programs list, you can reload the Home Connect integration when an appliance is turned on. If the re-initialization process is not triggered by reload, restart the Home Assistant when an appliance is turned on. 
```